src/Numerics/DGMethods/DGModel.jl

```diff
@@ -46,6 +46,27 @@ end
 # Include the remainder model for composing DG models and balance laws
 include("remainder.jl")
 
+function basic_grid_info(dg::DGModel)
+    grid = dg.grid
+    topology = grid.topology
+
+    dim = dimensionality(grid)
+    N = polynomialorder(grid)
+
+    Nq = N + 1
+    Nqk = dim == 2 ? 1 : Nq
+    Nfp = Nq * Nqk
+    Np = dofs_per_element(grid)
+
+    nelem = length(topology.elems)
+    nvertelem = topology.stacksize
+    nhorzelem = div(nelem, nvertelem)
+    nrealelem = length(topology.realelems)
+    nhorzrealelem = div(nrealelem, nvertelem)
+
+    return (Nq, Nqk, Nfp, Np, nvertelem, nhorzelem, nhorzrealelem, nrealelem)
```

Comment:
Maybe a `NamedTuple`?
```suggestion
    return (Nq=Nq, Nqk=Nqk, Nfp=Nfp, Np=Np, nvertelem=nvertelem, nhorzelem=nhorzelem, nhorzrealelem=nhorzrealelem, nrealelem=nrealelem)
```

Reply:
Done

Reply:
dammit, is there a way to enforce the formatter on committed suggestions :)